syntax = "proto3";

option go_package = "game/generated/pb/game";

import "proto/common/tip.proto";
import "proto/common/empty.proto";

option cc_generic_services = true;

message CentreKickPlayerRequest {
  TipInfoMessage reason = 1;
  string operator = 2;
}

service CentrePlayerUtility 
{
  rpc SendTipToClient (TipInfoMessage) returns (Empty);
  rpc KickPlayer (CentreKickPlayerRequest) returns (Empty);
}